AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -240,18 +240,15 @@ - (CGSize)calculateSizeThatFits:(CGSize)constrainedSize
   ASDN::MutexLocker l(__instanceLock__);
   CGSize calculatedSize = constrainedSize;
   
-  // if a preferredFrameSize is set, call the superclass to return that instead of using the image size.
-  if (CGSizeEqualToSize(self.preferredFrameSize, CGSizeZero) == NO)
-    calculatedSize = self.preferredFrameSize;
- 
   // Prevent crashes through if infinite width or height
   if (isinf(calculatedSize.width) || isinf(calculatedSize.height)) {
-    ASDisplayNodeAssert(NO, @"Infinite width or height in ASVideoNode");
+    //ASDisplayNodeAssert(NO, @"Infinite width or height in ASVideoNode");
```

Comment:
Yep this was temporary, but should be save to enable the assert again now. It's enabled again.